resources/eventing/charts/dashboards/templates/jetstream.yaml

```diff
@@ -101,7 +101,8 @@ data:
                    "refId":"A"
                 }
              ],
-             "title":"Storage Used",
+             "title":"File Storage Used (%)",
```

Comment:
nit:
the dial already says "%" so I am not sure if this is needed in the description too.

Reply:
There is another tile `File Storage Used` ([here](https://github.com/kyma-project/kyma/blob/dea1020ac29bc493e47697177258e5eaac5e0297/resources/eventing/charts/dashboards/templates/jetstream.yaml#L160)) which shows the actual amount used, so to differentiate with that tile I added the % sign.